test/integration/server0_p_10_p_1_test.exs

```diff
@@ -0,0 +1,45 @@
+defmodule KafkaEx.Server0P10P1.Test do
+  use ExUnit.Case
+  import TestHelper
+
+  @moduletag :server_0_p_10_p_1
+
+  @tag :createtopic
+  test "can create a topic" do
+    name = "create_topic_#{:rand.uniform(2000000)}"
+
+    request = %{
+      topic: name,
+      num_partitions: 10,
+      replication_factor: 1,
+      replica_assignment: [],
+      config_entries: [
+        %{config_name: "cleanup.policy", config_value: "compact"},
+        %{config_name: "min.compaction.lag.ms", config_value: "0"}
+      ]}
+
+    resp = KafkaEx.create_topics([request], timeout: 2000)
+    # error = NONE
+    assert {0, name} == parse_create_topic_resp(resp)
+
+    resp = KafkaEx.create_topics([request], timeout: 2000)
+    # error = TOPIC_ALREADY_EXISTS
+    assert {36, name} == parse_create_topic_resp(resp)
```

Comment:
We should be able to add 36 to the `@error_map` in `lib/kafka_ex/protocol.ex` and then refer to this as the error code (`:topic_already_exists`) rather than `36`

Reply:
Good idea, thanks ! I've added a few error codes, might as well :).